docs/buildstrategies.md

```diff
@@ -187,6 +187,16 @@ kubectl apply -f samples/buildstrategy/source-to-image/buildstrategy_source-to-i
 [s2i]: https://github.com/openshift/source-to-image
 [buildah]: https://github.com/containers/buildah
 
+## Replacement tokens
```

Comment:
I think a categorization will be needed soon. Not needed at the moment, as we move forward with all of this changes. Some ideas for categorizing params:

- System parameters: parameters which values are defined at runtime and that include SHP specifics. For example `shp-source-root` would be one of them because its `/workspace/source` is a SHP system specification. `shp-source-context` will be also one, because its value includes the `/workspace/source`prepended. The values of these parameters cannot be 100% defined by the Build API or strategy authors.

- Runtime parameters: parameters which values are define on runtime based purely on the Buid API fields values. 

- Parameters:  parameters which values are define on runtime based purely on the spec.params block.

Reply:
I changed to parameters now. This can then be extended once there are build-strategy-defined parameters.